xml/plivoxml.go

```diff
@@ -1139,7 +1139,7 @@ func ValidateLanguageVoice(language string, voice string) error {
 	}
 	transformedVoiceName := TransformString(voiceparts[1])
 
-	if strings.Compare(voiceparts[1], "*") == 0 || Contains(availableLanguageVoicesList, transformedVoiceName) == false {
+	if strings.Compare(voiceparts[1], "*") == 0 || Contains(availableLanguageVoicesList, transformedVoiceName) {
```

Comment:
shouldn't there be a not of Contains now, since we were comparing it with false earlier
`!Contains(availableLanguageVoicesList, transformedVoiceName)`

Reply:
Done